### DISCUSSION TITLE
I can't install GitHub Desktop

### DISCUSSION THREAD

**Discussion by YusukePlaysAPEX** [community/community#57248] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can't install GitHub Desktop on my PC as the picture shows.
How do you fix this??
![image](https://github.com/community/community/assets/93923404/0f63408b-aefb-478a-9bb9-8f63eeaedf17)

**Comment by YusukePlaysAPEX** [comment]
I didn't post any information cause I don't wanna leak my personal information.
If you want me to give some information to solve the problem, then I'll tell you.

**Comment by backtoaldo** [comment]
This error can be caused by problems installing the .NET Framework on your machine. On Microsoft's own website there is a link to a [.NET Repair Tool](https://www.microsoft.com/en-us/download/details.aspx?id=30135). Test and see if after repair, you are able to correctly install GitHubDesktop correctly.

**Reply by YusukePlaysAPEX** [reply]
@backtoaldo 
Thanks for your reply, but that didn't work🥲🥲
I'm using Japanese for the language of Windows, but does the tool still work on my PC?
![image](https://github.com/community/community/assets/93923404/7e3912e9-4c5d-4081-9bc7-8e5a31619ba0)

**Reply by YusukePlaysAPEX** [reply]
I mean I'm asking if the tool works even if the language is Japanese. 
I saw the language is English on the website but doesn't it depend on the repairment?

**Reply by backtoaldo** [reply]
According to Microsoft's own documentation, it should work correctly. [Link here](https://learn.microsoft.com/en-us/dotnet/framework/install/guide-for-developers).
![Captura de tela 2023-06-09 231223](https://github.com/community/community/assets/61122744/db277008-bc6e-461a-addb-e42ac9a52777)

**Reply by backtoaldo** [reply]
Follow[ this link](https://dotnet.microsoft.com/ja-jp/download/dotnet-framework/net481) and you will be able to download the pack you need.
![Captura de tela 2023-06-09 231459](https://github.com/community/community/assets/61122744/54217308-46eb-4bac-bea4-a62ad23f5079)
